pyproject.toml

```diff
@@ -36,6 +36,7 @@ dependencies = [
 
 [project.optional-dependencies]
 standard = [
+    "a2wsgi>=1.6.0,<2.0.0",
```

Comment:
Let's add `a2wsgi` as `uvicorn` dependency, and change the documentation as mentioned on https://github.com/encode/uvicorn/pull/1303#discussion_r803623891.

Notes about this decision:
- `a2wsgi` is [less than 15kB](https://pypi.org/project/a2wsgi/#files).
- `a2wsgi` is already the [recommended `WSGIMiddleware` on `Starlette`](https://github.com/encode/starlette/blob/master/starlette/middleware/wsgi.py).

Reply:
Yeah, is fine. 😊

Reply:
So the change will be the move from optional dependencies to the base dependecies, is that it?

Reply:
Yes, and to add on the documentation that Uvicorn is also a WSGI server.

Reply:
Et voilà :slightly_smiling_face: 

Since `WSGIMiddleware` converts WSGI into ASGI I described it as an `ASGI first web server` what do you think?